mod_wasm/mod_wasm.c

```diff
@@ -254,6 +262,59 @@ static void populate_runtime_with_config_defined_envs(x_cfg *cfg)
     }
 }
 
+
+
+/*
+ * This function reads the HTTP Request Body
+ * 
+ * r: request
+ * rbuff: buffer to where the body will be allocated
+ * size: size of the buffer allocated
+ * 
+ * More info:
+ *  - https://httpd.apache.org/docs/trunk/developer/modguide.html (section: "Reading the request body into memory")
+ *  - https://docstore.mik.ua/orelly/apache_mod/139.htm 
+ */
+static int read_http_request_body(request_rec *r, const char **rbuf, apr_off_t *size)
```

Comment:
I think we could get rid of the max body limitation if we follow mod_cgi's approach.

They call to `cgi_handle_request`. It reads the request body and writes it to a file descriptor. We could use this file descriptor to pass the body of the request to `wasi_stdin` instead of building an ad-hoc pipe.

I think it would be feasible, but haven't checked it.

Reply:
Good point @ereslibre!
Just created #23 to investigate this.